documentation/book/getting-started.adoc

```diff
@@ -59,6 +59,8 @@ endif::Kubernetes[]
 
 ==== Deploying to {OpenShiftName}
 
+IMPORTANT: to successfully deploy the Cluster Operator on {OpenShiftName} a user with `cluster-admin` role needs to be used (i.e. like `system:admin`).
```

Comment:
We should start with capital T ... _**T**o successfully ..._

Reply:
Done.